ci/libraries_tests/nrf5_app_usbd_hid_generic/main.c

```diff
@@ -1,12 +0,0 @@
-#include "app_usbd_hid_generic.h"
```

Comment:
Why removed?

Reply:
Actually, I cannot build this example due to a snipped of code you posted on Slack:
```C
CRITICAL_REGION_ENTER();
ret_code_t ret = app_usbd_ep_transfer(ep_addr, &transfer);
if (ret == NRF_SUCCESS)
{
    app_usbd_hid_state_flag_set(&p_generic_ctx->hid_ctx,
                                APP_USBD_HID_STATE_FLAG_TRANS_IN_PROGRESS);
}
CRITICAL_REGION_EXIT();

return ret;
```
The problem is, the `CRITICAL_REGION_ENTER/EXIT()` pair create an additional scope but only when we are building with `SOFTDEVICE_PRESENT`, which we do when building tests for libs. How did you fix that?

Reply:
Prepare script contains a patch for the SDK (look at ./ci/patches folder).